Add a note about hostname aliases to the FAQ (#17115)
DataDog/documentation: content/en/agent/faq/how-datadog-agent-determines-the-hostname.md

{{< img src="agent/faq/host_aliases.png" alt="Host aliases"  >}}

> **Note:** it's NOT possible to search or filter using these aliases, they are only available in the inspect panel mentioned above.

Reviewer: ```suggestion
**Note**: These aliases are not available for searching or filtering. They are available only in the inspect panel mentioned above.
```
